content/chapter-destinations/azure-monitor/_index.md

```diff
@@ -44,8 +49,8 @@ The body of the message (`${MESSAGE}`) must be in JSON format. The keys in the J
 
     ```sh
     destination d_azure_custom {
-      azure-monitor-custom(
-        table-name("my-table")
+      azure-monitor(
+        stream-name("my-table")
```

Comment:
We no longer need to mention "sending data into custom tables", so this example can be removed.
We require the user to set the stream name instead of the table name from now on, so there is no distinction between builtin and custom tables anymore.